New folder/lastmain.js

```diff
@@ -0,0 +1,103 @@
+// Ex1 returns an array with element elem repeated n times.
+const repeatElement = (element, repeat) => {
+  return Array(repeat).fill(element)
+}
+const repeatElementSecond = (element, repeat) => {
```

Comment:
2 block khác nhau thì cách nhau ra
```suggestion
}

const repeatElementSecond = (element, repeat) => {
```